### DISCUSSION TITLE
i don't know what is wrong

### DISCUSSION THREAD

**Discussion by KingofMidnight** [itzg/docker-minecraft-server#1807] [Q&A]
### Describe the problem

after i put FORCE_REDOWNLOAD: "true" into my docker-compose file and compose up after a long time this popped up

### Container definition

```
Paste run command or compose file here
```

version: "3.8"

services:
  mc:
    image: itzg/minecraft-server
    container_name: mc-6
    environment:
      SERVER_NAME: "PassiveMC"
      EULA: "TRUE"
      VERSION: "1.19" 
      MOTD: "Sup"
      DIFFICULTY: "hard"
      #INIT_MEMORY: "2G"
      MAX_MEMORY: "4G"
      ENABLE_RCON: "true"
      RCON_PASSWORD: "King"
      RCON_PORT: 25069
      ICON: ""
      OPS: 8b307bf1-ee01-49f0-b555-e586ca0baec7
      VIEW_DISTANCE: 12
      SIMULATION_DISTANCE: 12
      ENABLE_QUERY: "true"
      ENABLE_COMMAND_BLOCK: "true"
      USE_NATIVE_TRANSPORT: "true"
      TZ: "America/Puerto_Rico"
      FORCE_REDOWNLOAD: "true"
    ports:
      - "25565:25565"
    volumes:
      - data:/data
    stdin_open: true
    tty: true
    restart: unless-stopped
volumes:
  data: {}

### Container logs

```
Paste logs here
```

[init] Running as uid=1000 gid=1000 with /data as 'drwxrwxr-x 7 1000 1000 4096 Oct 28 18:29 /data'
[init] Resolved version given 1.19 into 1.19 and major version 1.19
[init] Resolving type given VANILLA
[init] Locating download for minecraft_server.1.19.jar ...
[mc-image-helper] 23:00:53.279 ERROR : 'get' command failed: Premature end of Content-Length delimited message body (expected: 45538778; received: 17055744)
[init] ERROR: failed to download server from https://launcher.mojang.com/v1/objects/e00c4052dac1d59a1188b2aa9d5a87113aaf1122/server.jar (1)

**Comment by itzg** [comment]
You have some kind of network issues on your system. 

I'm going to convert this to a discussion since the container image can't fix networking.
